CHANGELOG.md

```diff
@@ -8,19 +8,11 @@
 * SharedGroup::get_stats includes an optional parameter to get size of currently locked memory.
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
-* In cases where the main thread would exit before other threads, we could destroy a mutex
-  while it was still in use on other threads. This would cause a crash during shutdown.
-  This bug was introduced by https://github.com/realm/realm-core/pull/3185, which was part
-  of release 5.12.5
-
-### Breaking changes
-* None.
-
------------
-
-### Internals
-* None.
+* Fix a race during shutdown between access to and desctruction of a mutex, which could cause a crash.
```

Comment:
```suggestion
* When shutting down the server you could sometimes experience a crash with "realm::util::Mutex::lock_failed" in the stacktrace.
```

Reply:
But this should be moved to the public "Fixed" section?

Reply:
It is in the public section!

Reply:
Forgive me and my color blindness :-)
![image](https://user-images.githubusercontent.com/1215687/53089278-70365780-350c-11e9-8c0c-f6d71f562515.png)